.github/workflows/ci.yml

```diff
@@ -73,22 +71,21 @@ jobs:
           - "3.9"
           - "3.10"
         django-version:
-          - "3.2a"
-          - "4.0a"
+          - "3.2"
+          - "4.0"
         extra:
-          - ""
-          - "progressbar"
+          - "test"
+          - "test,progressbar"
     steps:
     - name: Set up Python ${{ matrix.python-version }}
       uses: actions/setup-python@v3
       with:
         python-version: ${{ matrix.python-version }}
     - run: sudo apt install gettext -y
     - uses: actions/checkout@v3
-    - run: python -m pip install --upgrade pip setuptools codecov wheel
-    - run: python -m pip install .[${{ matrix.extra }}]
+    - run: python -m pip install --upgrade pip
```

Comment:
If it could help, `codecov` is not installed anymore, could it be added again in order to run `codecov` or do you have another solution?

Reply:
Thx @jeromelebleu well spotted!